test/onnx/onnx_test_common.py

```diff
@@ -286,7 +286,7 @@ def run_test_with_fx_to_onnx_exporter_and_onnx_runtime(
             export_error = e
             onnx_program = e.onnx_program
 
-        if verbose and diagnostics.is_onnx_diagnostics_log_artifact_enabled():
```

Comment:
Do we still need verbose=True in run_test_xxxxx with this deleted here?

Reply:
after this pr we don't, I don't know why I put 'verbose' there before, feels only more inconvenient.

Reply:
I mean we could delete it from this function?

Reply:
you are right, good catch.